reports/3_generate_report_emails.py

```diff
@@ -0,0 +1,145 @@
+# +
+from calitp.tables import tbl
+from siuba import *
+import sys
+from postmarker.core import PostmarkClient
+
+import pandas as pd
+
+
```

Comment:
Either add the token as a commandline argument (e.g. pull from sys.argv), or ideally use os

e.g.

```python
import os
SERVER_TOKEN = os.environ["POSTMARK_SERVER_TOKEN"]
```